app/internal/wysiwyg_editor.py

```diff
@@ -0,0 +1,44 @@
+import bleach
+
+
+def clean_html(html: str) -> str:
+    """
+    This function gets an html string  and checks
+    for malicious and not allowed tags, attrs and styles
+    and then returns it cleaned without the not allowed parameters.
+
+    Args:
+        html (str): string from a post request made by the summernote editor.
+
+    Returns:
+        str: bleached string of html.
+    """
+
+    allowed_tags = [
+        'a', 'abbr', 'b', 'blockquote', 'br', 'strike', 'u',
+        'code', 'dd', 'del', 'div', 'dl', 'dt', 'em',
+        'em', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'hr', 'i', 'img', 'li',
+        'ol', 'p', 'pre', 's', 'strong', 'sub', 'sup',
+        'table', 'tbody', 'td', 'th', 'thead', 'tr', 'ul', 'span', 'font'
```

Comment:
`font` and `img` are problematic, it allows information leakage. Can't the WYSIWYG editor return markdown instead of HTML?

Reply:
https://github.com/summernote/summernote/projects/2
it's in their to-do list.
Would you rather not let people use img and font?

Reply:
TBH, I find it scary as hell to allow plain HTML in our system.
It's in their TDL from 2018.

How about using https://github.com/nhn/tui.editor?

Reply:
changed to tui.editor